gradio/blocks.py

```diff
@@ -831,6 +762,7 @@ async def call_function(
         processed_input: List[Any],
         iterator: Iterator[Any] | None = None,
         request: routes.Request | List[routes.Request] | None = None,
+        event_id: str | None = None,
     ):
         """Calls and times function with given index and preprocessed input."""
```

Comment:
Even though this is an internal function, it would be super helpful if we could add docstring to explain the parameters. Would you be able to add?